airbyte-workers/src/main/java/io/airbyte/workers/protocols/airbyte/AirbyteDestination.java

```diff
@@ -4,9 +4,78 @@
 
 package io.airbyte.workers.protocols.airbyte;
 
+import io.airbyte.commons.functional.CheckedConsumer;
+import io.airbyte.config.WorkerDestinationConfig;
 import io.airbyte.protocol.models.AirbyteMessage;
-import io.airbyte.workers.protocols.Destination;
+import java.nio.file.Path;
+import java.util.Optional;
 
-public interface AirbyteDestination extends Destination<AirbyteMessage> {
+/**
+ * This interface provides a java interface over all interactions with a Destination from the POV of
+ * the platform. It encapsulates the full lifecycle of the Destination as well as any inputs and
+ * outputs.
+ */
+public interface AirbyteDestination extends CheckedConsumer<AirbyteMessage, Exception>, AutoCloseable {
+
+  /**
+   * Starts the Destination container. It instantiates a writer to write to STDIN on that container.
+   * It also instantiates a reader to listen on STDOUT.
+   *
+   * @param destinationConfig - contains the arguments that must be passed to the read method of the
```

Comment:
```suggestion
   * @param destinationConfig - contains the arguments that must be passed to the write method of the
```
Should this say `write method` here?